addons/sale/views/sale_portal_templates.xml

```diff
@@ -554,6 +554,7 @@
                 <div id="total" class="row" name="total" style="page-break-inside: avoid;">
                     <div t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'} ms-auto">
                         <t t-call="sale.sale_order_portal_content_totals_table"/>
+                        <t t-call="sale.sale_order_portal_content_loyalty_table"/>
```

Comment:
This should be done using xpath extensions in `sale_loyalty` rather than in the base module.